mobile/assets/i18n/en.json

```diff
@@ -1,6 +1,16 @@
 {
     "newBooks": "New Books for You",
     "allBooks": "All Books",
+    "letsTalk": "Let's Talk",
+    "tel": "Tel: ",
+    "num": "858.274.9673",
+    "fax": "Fax: ",
+    "days": "Mon-Fri: ",
+    "hours": "8am-4pm",
```

Comment:
Oh, I think we might be able to do this automatically! https://github.com/fnando/i18n-js#date-formatting